mycpp/visitor.py

```diff
@@ -235,12 +239,17 @@ def oils_visit_class_def(
             if isinstance(stmt, FuncDef):
                 method_name = stmt.name
 
+                if self.current_class_name in self.methods:
+                    self.methods[self.current_class_name].append(method_name)
+                else:
+                    self.methods[self.current_class_name] = [method_name]
```

Comment:
Hm what if b() is defined before a() ?  I think we might be accumulating them in order here